corehq/apps/hqmedia/views.py

```diff
@@ -490,6 +490,7 @@ def log_errors(self, errors):
         )
 
     @method_decorator(safe_download)
+    @method_decorator(require_permission(Permissions.edit_apps, login_decorator=login_with_permission_from_post()))
```

Comment:
does the phone never hit this URL? that's one situation we have to be careful of.